apps/bfd-pipeline/bfd-pipeline-rda-grpc-apps/src/main/java/gov/cms/bfd/pipeline/rda/grpc/apps/LoadRdaJsonApp.java

```diff
@@ -44,8 +44,15 @@
  * data so either or both can be provided as needed.
  */
 public class LoadRdaJsonApp {
-  private static final Logger logger = LoggerFactory.getLogger(LoadRdaJsonApp.class);
+  private static final Logger LOGGER = LoggerFactory.getLogger(LoadRdaJsonApp.class);
```

Comment:
no big deal...just curious on changing case on logger; I guess to match BFD name?

Reply:
LOGGER is recognized by default by java docs :)

Reply:
Static final vars should be uppercase by convention. Checkstyle has an exception for such loggers (properly named) since theres basically never anything to say about the logger.